cdmtaskservice/jaws/wdl.py

```diff
@@ -67,7 +67,7 @@ def generate_wdl(
         input_files.append(ins)
         relpaths.append(rels)
         cmd = [shlex.quote(c) for c in job.image.entrypoint]
-        # TODO handle flag args
+        cmd.extend(_process_flag_args(job, files, file_to_rel_path))
```

Comment:
`determine_file_locations` returns `dict[S3File | str, str]` but `_process_flag_args` expects `dict[S3File, Path]`.

Reply:
Good eye. I've got this fixed locally, it'll be in a future PR